src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -2190,3 +2190,78 @@ void cprLookupByAttribute(EntityId&                          en,
     }
   }
 }
+
+#define DOT_ESCAPE_CHAR '='
+
+/* ****************************************************************************
+*
+* dbDotEncode -
+*
+* Replace:
+*   . => =
+*/
+std::string dbDotEncode(std::string fromString)
```

Comment:
You told me about this function could be simplified removing malloc(), etc. However, I prefer that you do the exact changes, as you this stuff better than me :). To be practical, if you don't want to change branch, recompile, etc. in your local environment, add the code in a comment here and I will copy-paste it to the .cpp file.